rpm-ostree/playtron-os.yaml

```diff
@@ -88,22 +95,28 @@ packages:
   - xorg-x11-xinit
   ## Required for Playserve.
   - xorg-x11-server-Xvfb
+  ## Nvidia support
```

Comment:
Do we want to remove all these commented out packages?

Reply:
Good call! Yeah, that basically never worked. I'll go ahead and remove it.